spec/lib/secure_headers/headers/content_security_policy_spec.rb

```diff
@@ -141,6 +141,11 @@ module SecureHeaders
         expect(csp.value).to eq("default-src 'self'; require-sri-for script style")
       end
 
+      it "allows style as a require-trusted-types-src" do
```

Comment:
Nit: I don't think `require-trusted-types-src` is defined?
```suggestion
      it "allows style as a require-trusted-types-for source" do
```

Reply:
oof, you're right. thanks!